CMakeLists.txt

```diff
@@ -182,7 +185,10 @@ else()
 
     # Arch Options
     if(ARCH STREQUAL "i386")
-        add_definitions(-D_M_IX86 -D_X86_ -D__i386__ -Di386)
+        if(NOT USE_CLANG_CL)
+            add_definitions(-D_M_IX86)
```

Comment:
This breaks clang?

Reply:
They have a built-in version so ours counts as a redefinition.